src/main/java/io/ebeaninternal/server/querydefn/DefaultOrmQuery.java

```diff
@@ -1220,6 +1221,10 @@ public void findEach(Consumer<T> consumer) {
   @Override
   @SuppressWarnings("unchecked")
   public <A> List<A> findSingleAttributeList() {
+    if (!detail.hasSelectClause()) {
+      // (no explicit select set - clear all properties)
+      detail.setBase(new OrmQueryProperties(null, new LinkedHashSet<>()));
+    }
     return (List<A>) server.findSingleAttributeList(this, null);
```

Comment:
This does not look right the correct place for this if ... setBase() code .... in that people can call server.findSingleAttributeList(Query, Transaction) and this check should also support that alternative way to execute the query. 

I'll look to move this check after merge.